engines/dfc_provider/app/controllers/dfc_provider/api/v0/base_controller.rb

```diff
@@ -4,7 +4,9 @@
 module DfcProvider
   module Api
     module V0
-      class BaseController < ActionController::Base
+      class BaseController < ::Api::V0::BaseController
+        skip_authorization_check
```

Comment:
It's a bit confusing that you first skip authorisation and then you add it below. Maybe your own check should be renamed to `check_access_token` to avoid using the same method name to do different things.

Reply:
Ah yes, actually it is 2 different methods but I agree it is confusing, I will rename my method.